templates/main.html

```diff
@@ -182,9 +194,14 @@ <h3>
       </a>
     </div>
     <footer>
-      <small>Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small>
+      <small data-i18n="footer-text">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small>
     </footer>
-    <script src="js/app.js"></script>
+    <script>
+      $(window).on('load', function() {
+        $.getScript('js/app.js')
```

Comment:
I saw you were using `defer` before, why change?

Reply:
`defer` was not working properly. I implemented this workaround to make sure that the translation file finished importing before applying the translations

Reply:
>translation file finished importing

Then it's not `$(window).on('load')` then, there must be another event from jquery.i18n

Reply:
It needs to bc the text to translate (the HTML) needs to load as well so `$(window).on('load')` is the correct way.